web/src/features/map-controls/ZoomControls.tsx

```diff
@@ -1,12 +1,15 @@
+import { useFeatureFlag } from 'features/feature-flags/api';
 import { ReactElement } from 'react';
 import { NavigationControl } from 'react-map-gl/maplibre';
 
 export default function ZoomControls(): ReactElement {
+  const isConsumptionOnlyMode = useFeatureFlag('consumption-only');
+  const marginTop = isConsumptionOnlyMode ? 54 : 98;
```

Comment:
QQ: Where do these numbers come from?

Reply:
I think it's the absolute positioning the zoom buttons use. These comes from the Maplibre css file and the maplibre built-ins. I tried to fix it in #6974 but then broke the wind layer instead (which should be unrelated but apparently is not).